e2e/cypress/support/compliance_export_commands.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../fixtures/timeouts';
+
+Cypress.Commands.add('enableComplianceExport', (exportFormate = 'csv') => {
+    // # Enable compliance export
+    cy.findByRole('radio', {name: /true/i}).click();
+
+    // # Change export format
+    cy.findByRole('combobox', {name: /export format:/i}).select(exportFormate);
+
+    // # Save settings
+    cy.findByRole('button', {name: /save/i}).as('saveButton');
+    cy.get('@saveButton').click();
+
+    cy.waitUntil(() => cy.get('@saveButton').then((el) => {
+        return el[0].innerText === 'Save';
+    }));
+});
+
+Cypress.Commands.add('gotoCompliancePage', () => {
+    cy.visit('/admin_console/compliance/export');
+    cy.get('.admin-console__header', {timeout: TIMEOUTS.TWO_MIN}).should('be.visible').invoke('text').should('include', 'Compliance Export');
+});
+
+Cypress.Commands.add('exportCompliance', () => {
```

Comment:
Please rename, move to `e2e/cypress/support/ui/compliance_export.js` and add corresponding entry to `e2e/cypress/support/ui/compliance_export.d.ts`
```suggestion
Cypress.Commands.add('uiExportCompliance', () => {
```